core/src/mindustry/ui/fragments/BlockInventoryFragment.java

```diff
@@ -142,7 +142,7 @@ private void rebuild(boolean actions){
 
                 container.add(i);
 
-                Boolp canPick = () -> player.unit().acceptsItem(item) && !state.isPaused() && player.within(tile, itemTransferRange);
```

Comment:
This means that the cursor will still display even when the player doesn't have room for the item. Not desirable.